waku/waku_relay/protocol.nim

```diff
@@ -216,19 +209,38 @@ proc generateOrderedValidator*(w: WakuRelay): auto {.gcsafe.} =
     let msg = msgRes.get()
 
     # now sequentially validate the message
-    for validator in w.wakuDefaultValidators:
+    for (validator, _) in w.wakuValidators:
       let validatorRes = await validator(pubsubTopic, msg)
       if validatorRes != ValidationResult.Accept:
         return validatorRes
-
-    if w.wakuValidators.hasKey(pubsubTopic):
-      for validator in w.wakuValidators[pubsubTopic]:
-        let validatorRes = await validator(pubsubTopic, msg)
-        if validatorRes != ValidationResult.Accept:
-          return validatorRes
     return ValidationResult.Accept
   return wrappedValidator
 
+proc isValidSize(message: WakuMessage): Future[Result[void, string]] {.async.} =
```

Comment:
I'll also add to this proc a check that the message size is smaller than GossipSub's max.

Currently the max size is defined in `nim-libp2p` here: https://github.com/status-im/nim-libp2p/blob/e3c967ad1939fb33b8e13759037d193734acd202/libp2p/protocols/pubsub/pubsub.nim#L556

I don't want to have it hardcoded, so I'll first open a PR in `nim-libp2p` defining it as a global `const` and will then add an `if` condition in this proc